linter_plugin.py

```diff
@@ -1,31 +1,50 @@
-"""Certbot ACME PyLint plugin.
-
+"""Certbot PyLint plugin.
 http://docs.pylint.org/plugins.html
-
 """
-from astroid import MANAGER
-from astroid import nodes
+from pylint.checkers import BaseChecker
+from pylint.interfaces import IAstroidChecker
+from pylint.checkers.utils import check_messages
```

Comment:
I think we should remove this. It doesn't seem to be documented for use in custom checkers and looking at the code, I'm not sure how using the decorator helps us. If you have a different understanding of its purpose, please let me know, but it seems the [decorator](https://github.com/PyCQA/pylint/blob/293a7f5e6add850e2f127dd1680a6e43fb1a6949/pylint/checkers/utils.py#L430) just gives the functions a `checks_msgs` attribute which is used [here](https://github.com/PyCQA/pylint/blob/293a7f5e6add850e2f127dd1680a6e43fb1a6949/pylint/utils/ast_walker.py#L19). This allows pylint to skip running functions for disabled messages, however, I think we always want this check enabled so I think using the decorator adds slightly more complexity for no real gain.

Reply:
You are right. I remove it.